### PR TITLE
Japanese translation of Code of Coduct

### PR DESCRIPTION
**Pull request by NaokiMatsumoto709**
Addresses #15

### CODE REVIEW

File: code-of-conduct-JP.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+## 行動規範
```
Comment (AkihiroSuda) [inline code-of-conduct-JP.md]: "行動規範"→"はじめに"?
(original text: "Introduction")

File: code-of-conduct-JP.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+我々は事業全般を通じ、誰もがハラスメント行為を受けることのない職場環境を作り、いかなる形のハラスメント、尚且つそれの参加者を容認することをしない。
```
Comment (AkihiroSuda) [inline code-of-conduct-JP.md]: I'm not sure the original text is limited to "職場環境" (office) and "事業" (enterprise).

My suggestion is "我々は，ハラスメントのない体験を，すべての人に提供することに徹し，...". WDYT?

(original text: "We are dedicated to providing a harassment­ free experience for everyone, and we do not tolerate harassment of participants in any form.")

File: code-of-conduct-JP.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+我々の行動規範として、社員一人一人が他者を尊重し、プロとしての意識をもって行動することを必要とする。
```
Comment (AkihiroSuda) [inline code-of-conduct-JP.md]: "社員"(employee) → "参加者"(participant) ?
(original text: "We ask you to be considerate of others and behave professionally and respectfully to all other participants.")

File: code-of-conduct-JP.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+この規範は社外での活動やその他全面―オンライン、現実を問わず―かつ一対一でのコミュニケーションの際におこる不適切な行動においても適応され、その行動がこのコミュニティーの一員の安全に関わるどの場に置いても適応される。
```
Comment (AkihiroSuda) [inline code-of-conduct-JP.md]: "社外" (outside the company) → "コミュニティ活動の範囲外" (outside the scope of community activities)?

(original text: "This code and related procedures also apply to unacceptable behavior occurring outside the scope of community activities, in all community venues— online and in-person— as well as in all one-on-one communications, and anywhere such behavior has the potential to adversely affect the safety and well-being of community members.")

File: code-of-conduct-JP.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+出展者、演説者、スポンサー、スタッフなどのDocker, Incのイベントの参加者、また、Docker, Incの施設内で行われる行事の参加者はこの行動規範の対象となる。
+Dockerコミュニティは多様性を理解し、多種多様な人材の受け入れを推進している。我々は多様多彩な経歴の持ち主の導入を可能な限り支持し、我々の立場を明確にするよう努力している。
```
Comment (AkihiroSuda) [inline code-of-conduct-JP.md]: Translation for "Diversity and inclusion make the Docker community strong." seems missing

File: code-of-conduct-JP.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+- 性、性同一性、性表現、性的指向、障害、身体的外見、身体の大きさ、人種、民族性、国籍、宗教、年齢などに関する攻撃的、不適切、または不必要なコメント。
+- 視的嫌がらせ。（例）Dockerコミュニティ行事での性的画像、性的言語の使用
+- 他者の意見への軽視。
+- 意図的脅迫、ストーカー行為、撮影や録画による嫌がらせ
+- 会話やイベント行事への一貫した妨害
+- 不適切な身体的接触
+- 他者への脅迫または虐め―オンライン、現実を問わず―
+- 性的嫌がらせ
+
+## 社内報告及び執行
```
Comment (AkihiroSuda) [inline code-of-conduct-JP.md]: "社内報告" (internal reporting) → "報告" (reporting)?

File: code-of-conduct-JP.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+- 親切心を持つこと。
+- 他者の見解や考えを尊重すること。
+- 他者を支持し、協力すること。
+
+## 不適切な行動
+
+ハラスメント行為の一例:
+- 性、性同一性、性表現、性的指向、障害、身体的外見、身体の大きさ、人種、民族性、国籍、宗教、年齢などに関する攻撃的、不適切、または不必要なコメント。
+- 視的嫌がらせ。（例）Dockerコミュニティ行事での性的画像、性的言語の使用
+- 他者の意見への軽視。
```
Comment (AkihiroSuda) [inline code-of-conduct-JP.md]: "軽視" (consider less important) -> "軽蔑" (disrespect)?

File: code-of-conduct-JP.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+- 意図的脅迫、ストーカー行為、撮影や録画による嫌がらせ
+- 会話やイベント行事への一貫した妨害
+- 不適切な身体的接触
+- 他者への脅迫または虐め―オンライン、現実を問わず―
```
Comment (AkihiroSuda) [inline code-of-conduct-JP.md]: "現実"(reality) → "対面" (in-person)

"オンライン"(online) is "現実"(reality) as well 😅

File: code-of-conduct-JP.md (hunk 5 above)
Comment (NaokiMatsumoto709) [inline code-of-conduct-JP.md]: I changed the translation.  WDYT?

**Comment by NaokiMatsumoto709** [comment]
@AkihiroSuda Thank you for your help, I've incorporated your suggestions.  Please take a look at the updates.

File: code-of-conduct-JP.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
+## 初めに
```
Comment (AkihiroSuda) [inline code-of-conduct-JP.md]: usually spelled in Hiragana

File: code-of-conduct-JP.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+
+我々は，ハラスメントのない体験を，すべての人に提供することに徹し、いかなる形のハラスメント、尚且つそれの参加者を容認することをしない。
+我々の行動規範として、参加者一人一人が他者を尊重し、プロとしての意識をもって行動することを必要とする。
+この規範はコミュニティ活動の範囲外やその他全面―オンライン、対面を問わず―かつ一対一でのコミュニケーションの際におこる不適切な行動においても適応され、その行動がこのコミュニティーの一員の安全に関わるどの場に置いても適応される。
+
+出展者、演説者、スポンサー、スタッフなどのDocker, Incのイベントの参加者、また、Docker, Incの施設内で行われる行事の参加者はこの行動規範の対象となる。
```
Comment (AkihiroSuda) [inline code-of-conduct-JP.md]: translation for "(DockerCon, meetups, user groups)" seems missing

Comment (AkihiroSuda) [inline code-of-conduct-JP.md]: and "Community Guidelines"

File: code-of-conduct-JP.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+多様性の理解、多種多様な人材の受け入れはDockerコミュニティをより強力なものとしてる。我々は多様多彩な経歴の持ち主の導入を可能な限り支持し、我々の立場を明確にするよう努力している。
```
Comment (AkihiroSuda) [inline code-of-conduct-JP.md]: してる -> している or する

Comment (AkihiroSuda) [inline code-of-conduct-JP.md]: "多様性と包摂" would be closer to the original text than "多様性の理解、多種多様な人材の受け入れ"

Comment (AkihiroSuda) [inline code-of-conduct-JP.md]: 導入 -> 参加 (original text: participation)

File: code-of-conduct-JP.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
+## はじめに
+
+我々は，ハラスメントのない体験を，すべての人に提供することに徹し、いかなる形のハラスメント、尚且つそれの参加者を容認することをしない。
```
Comment (AkihiroSuda) [inline code-of-conduct-JP.md]: することをしない -> しない

**Comment by thaJeztah** [comment]
ping @AkihiroSuda looks like this was updated, PTAL :heart:

**Comment by thaJeztah** [comment]
Thanks @NaokiMatsumoto709, @AkihiroSuda !

**Pull request merged**